checks/http/cambium.rb

```diff
@@ -0,0 +1,41 @@
+module Intrigue
+    module Ident
+    module Check
+    class Cambium < Intrigue::Ident::Check::Base
+    
+      def generate_checks(url)
+        [
+          {
+            type: "fingerprint",
+            category: "hardware",
+            tags: ["Embedded", "Networking"],
+            vendor:"Cambium",
+            product: nil,
+            description: "page title",
+            match_type: :content_body,
+            match_content:  /<title>ePMP/,
+            version: nil, 
+            paths: [ { path: "#{url}", follow_redirects: true } ],
+            inference: false
+          },
+          {
+            type: "fingerprint",
+            category: "hardware",
+            tags: ["Embedded", "Networking"],
+            vendor:"Cambium",
+            product: nil,
```

Comment:
We always go with NVD naming whenever possible, as - when we get a specific enough version - the inference: attribute tells ident to match vulns on version.. 

NVD doesnt split out the http server, so i think what you've done here is perfect. https://nvd.nist.gov/products/cpe/search/results?namingFormat=2.3&keyword=cambium